.travis.yml

```diff
@@ -46,7 +44,7 @@ env:
   - RUN_TEST_GROUP=coverage
 
 jdk:
-  - oraclejdk8
+  - openjdk8
```

Comment:
Curious: can we make do without this change?

Reply:
xenial wanted oraclejdk9+
by chance it accepted openjdk8 though.
we should start supporting jdk9+ also, but i found a small portion of gobblin code is not compatible with jdk9

Reply:
Ok. I am not opinionated about this. Would like to hear what others think @htran1 ?

Reply:
Other option is to remain at trusty , but do not use pre installed mysql.
My opinion is that we should make efforts to support newer version of OS and java.